draft-packages/collapsible/KaizenDraft/Collapsible/styles.scss

```diff
@@ -29,27 +31,30 @@ $heading-active-color: $kz-var-color-wisteria-100;
   }
 }
 
+// When a collapsible group is rendered, we need the first group to have a rounded
+// on top and the last group to have a rounded bottom edge. Then when open, we
+// remove these as the content sits beneath and needs to be straight.
 .groupItem {
   & + .groupItem {
     border-top: 1px solid $ca-border-color;
   }
 
-  &:first-of-type > .button {
+  &:first-of-type > .button:not(.open) {
```

Comment:
I don't think `:not(.open)` will work here. It results in the top border radius only being set when the collapsible is closed. Meaning the corners stick out when `.open` is set.

Reply:
Ah bugger, I was trying to get a nicer solution than another nested declaration. Will fix this up.